packages/application-shell/src/components/redirect-to-project-create/redirect-to-project-create.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { Spacings, Text, LinkButton } from '@commercetools-frontend/ui-kit';
+
+export const RedirectToProjectCreate = props => {
+  if (props.servedByProxy === true)
+    window.location.replace('/account/projects/new');
```

Comment:
Is it "safe" to do this during rendering? 🤔
I mean, the page reloads so who cares (or maybe sentry cares), but in theory we should do that using a side effect, correct?

Reply:
If we do it as a side-effect it will happen after the component has been mounted. Hence you will see the message then be redirected. If we do it in render if occurs faster. I also don't like side-effects in render but in this case we just want to not even work with anything rendered anymore.